frappe/public/js/frappe/model/meta.js

```diff
@@ -192,6 +192,15 @@ $.extend(frappe.meta, {
 		}
 	},
 
+	get_print_sizes: function() {
```

Comment:
Feels like this should be fetched from Print Settings' pdf_page_size field's options instead. Or linked in some way. 🤔

Reply:
Oh, that's a good point.

Is there a way I can fetch field options from JS, or does this need to be a database call?

Reply:
yep, either database call (to load meta) or we'll have to load it via `frappe.boot`.  The current implementation seems fine since these sizes will remain constant.